src/containers/tutor-profile/profile-info/ProfileInfo.jsx

```diff
@@ -54,6 +53,42 @@ const ProfileInfo = () => {
     </IconButton>
   )
 
+  const accountRating = (
+    <>
+      <StarIcon sx={styles.ratingIcon} />
+      {userData.averageRating.tutor}
+    </>
+  )
+
+  const linkToReviews = (
+    <Typography
+      component={HashLink}
+      sx={{ color: 'text.primary' }}
+      to={'#'}
+      variant='overline'
+    >
+      {`${userData.totalReviews.tutor} reviews`}
+    </Typography>
+  )
+
+  const accountInfoMock = [
+    {
+      title: `${getNumberOfYears(
+        userData.createdAt,
+        userData.updatedAt
+      )} years`,
+      description: 'at space2study'
+    },
+    {
+      title: accountRating,
+      description: linkToReviews
+    },
+    {
+      title: '75 UAH',
+      description: 'hourly rate'
```

Comment:
take this text from translation too, it is already there

Reply:
Fixed.